### PR TITLE
Docs update

### PR DESCRIPTION
**Pull request by AdamKufel**
Minor updates to docs

### CODE REVIEW

File: docs/docker.md [hunk 1]
```diff
@@ -23,42 +21,39 @@ Quick Start:
...
-You can make additional Docker configuration changes by modifying `docker-compose.init.yml`. Additionally, before you bring the containers up,
-you need to add an account for the scheduler to monitor. This can all be done one of two ways:
-    1. By modifying the `docker/api-init.sh` script to include the `monkey` commands. **PLEASE NOTE:** If you make changes to this file, 
-       please run `docker-compose down && docker-compose build` to prevent errors and potential annoyances.
-    1. By manually logging into the data container's shell to execute the commands.
+1.  On a fresh database instance, various initial configuration must be run such as database setup, initial user creation (<admin@example.org> / admin), etc.
+    You can make additional Docker configuration changes by modifying `docker-compose.init.yml`. Additionally, before you bring the containers up,
+    you need to add an account for the scheduler to monitor. This can all be done one of two ways: 1. By modifying the `docker/api-init.sh` script to include the `monkey` commands. **PLEASE NOTE:** If you make changes to this file,
```
Comment (mikegrima) [inline docs/docker.md]: The numbers need to be on their own lines.

**Review by mikegrima — CHANGES_REQUESTED** [review]
The only change in the doc required is:
`    $ monkey add_account_aws --id $account --name $name --role_name SecurityMonkey`

The connection string does not contain a typo, `postgresql` is correct. The other changes are not needed.

**Comment by AdamKufel** [comment]
Indeed it contains a typo, in documentation (https://github.com/Netflix/security_monkey/blob/develop/docs/options.md) it's stated: 

> "SQLALCHEMY_DATABASE_URI = 'postgressql://\<user\>:\<password\>@\<hostname\>:5432/SecurityMonkey'"

there is "postgre**ss**ql" (double "ss") but correct protocol is "postgre**s**ql" (one "s")

**Comment by mikegrima** [comment]
Sorry -- I misread.

Can you update your PR to only make those really small updates? Some of the numbering formatting is broken in the current PR.

**Pull request closed without merging**